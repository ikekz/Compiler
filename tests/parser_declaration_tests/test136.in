function f(a: integer) : integer;
var
	a: integer;
begin
end;
begin
end.