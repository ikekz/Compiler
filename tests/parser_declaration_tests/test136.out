(3,2) Error: Duplicate identifier "a"